cmd/jimmsrv/service/service_test.go

```diff
@@ -36,12 +38,33 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+// newTestJimmParams returns a set of JIMM params with sensible defaults
+// for tests. A test can override any parameter that it needs.
+// Note that newTestJimmParams will create an empty test database.
+func newTestJimmParams(t jimmtest.Tester) jimmsvc.Params {
```

Comment:
why not exporting this function and using it all around? As service_test configuration?

Reply:
ignore my comment